certified-connectors/InEight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3050 @@
+          "text/plain"
+        ],
+        "produces": [
+          "application/json",
+          "text/plain"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/Crafts/Import": {
+      "post": {
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "type": "string"
+            },
+            "headers": {
+              "cache-control": {
+                "description": "cache-control",
+                "type": "string"
+              },
+              "content-length": {
+                "description": "content-length",
+                "type": "string"
+              },
+              "content-type": {
+                "description": "content-type",
+                "type": "string"
+              },
+              "date": {
+                "description": "date",
+                "type": "string"
+              },
+              "expires": {
+                "description": "expires",
+                "type": "string"
+              },
+              "location": {
+                "description": "location",
+                "type": "string"
+              },
+              "pragma": {
+                "description": "pragma",
+                "type": "string"
+              },
+              "request-context": {
+                "description": "request-context",
+                "type": "string"
+              },
+              "retry-after": {
+                "description": "retry-after",
+                "type": "string"
+              },
+              "x-ms-apihub-cached-response": {
+                "description": "x-ms-apihub-cached-response",
+                "type": "string"
+              },
+              "x-powered-by": {
+                "description": "x-powered-by",
+                "type": "string"
+              },
+              "x-requestid": {
+                "description": "x-requestid",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Crafts_Import",
+        "description": "Crafts Integration allows InEight customers to provide a list of crafts for their organization.  Within the application, Crafts records are found in the Account level menu options under  \"Master data libraries > Operational resources\" or from within the Project level menu under  \"Assigned operational resources\".",
+        "operationId": "Crafts_Import",
+        "parameters": [
+          {
+            "name": "X-IN8-TENANT-PREFIX",
+            "in": "header",
+            "description": "The tenant-prefix of your Project Suite base URL. For example, if you are accessing Project Suite at *https://sample-domain.hds.ineight.com/* then the value for this parameter would be **sample-domain**",
+            "x-ms-summary": "The tenant-prefix of https://t01.hds.ineight.com/ would be \"t01\"",
```

Comment:
@InEightDevOps It seems a trademark unicode. If yes, adding copyright or register or trademark symbols in any connector file is not allowed, Our legal team will not let us certify with these symbols anywhere in the connector artifacts or files. Please remove them everywhere in your connector files.